docs/source/index.rst

```diff
@@ -4,7 +4,7 @@ Welcome to AiiDA's documentation!
 
 **aiida-core version:** |release|
 
-`AiiDA`_ is an open-source framework implemented in Python that aims to help researchers with managing simple to complex workflows and making them fully reproducible (see :ref:`features<intro:about>`).
+`AiiDA`_ is an open-source Python infrastructure to automate, manage, persist, search and reproduce the complex workflows associated with modern computational science, and their associated data. (see :ref:`features<intro:about>`).
```

Comment:
```suggestion
`AiiDA`_ is an open-source Python infrastructure to automate, manage, persist, search and reproduce the complex workflows and their associated data associated with modern computational science. (see :ref:`features<intro:about>`).
```

Reply:
Do we need both "automate" and "manage". I have the feeling that one of the two should accurately cover the meaning of both

Reply:
I generally don't like the style of the rephrased version, but if we do change it I'd propose `and all associated` instead of `and their associated data`.

Reply:
I think manage conveys more the meaning of "organise". I applied Simon's suggestion, and rephrased a bit to be "midway" between the old and the new. Let me know if it is now ok.